terraform/services/quicksight/firehose-agg.tf

```
@@ -0,0 +1,109 @@
+# Firehose Data Stream
+resource "aws_kinesis_firehose_delivery_stream" "firehose-ingester-agg" {
```

The resource name doesn't need to repeat the resource, and we should stick to underscores:

```suggestion
resource "aws_kinesis_firehose_delivery_stream" "ingester_agg" {
```